notifier.rst

```diff
@@ -115,12 +115,18 @@ The chat channel is used to send chat messages to users by using
 :class:`Symfony\\Component\\Notifier\\Chatter` classes. Symfony provides
 integration with these chat services:
 
-========  =============================  ============================================
-Service   Package                        DSN
-========  =============================  ============================================
-Slack     ``symfony/slack-notifier``     ``slack://TOKEN@default?channel=CHANNEL``
-Telegram  ``symfony/telegram-notifier``  ``telegram://TOKEN@default?channel=CHAT_ID``
-========  =============================  ============================================
+==========  ===============================  ============================================
+Service     Package                          DSN
+==========  ===============================  ============================================
+Slack       ``symfony/slack-notifier``       ``slack://TOKEN@default?channel=CHANNEL``
+Telegram    ``symfony/telegram-notifier``    ``telegram://TOKEN@default?channel=CHAT_ID``
+Mattermost  ``symfony/mattermost-notifier``  ``mattermost://ENDPOINT?channel=CHANNEL``
+RocketChat  ``symfony/rocketchat-notifier``  ``rocketchat://ENDPOINT?channel=CHANNEL``
```

Comment:
Sorry to be too late to this party, but the DSN can be a bit more specific (as done for the other DSNs). See the [`xxxTransportFactory::create($dsn)`](https://github.com/symfony/symfony/pull/34022/files#diff-befa7e13b0d3c6096c1197cae153afcfR26-R39) methods of what to expect:

RocketChat: `rocketchat://TOKEN@ENDPOINT?channel=CHANNEL`
Mattermost: `mattermost://TOKEN@ENDPOINT?channel=CHANNEL`